single_kernel_mongo/managers/mongos_operator.py

```diff
@@ -190,14 +211,17 @@ def on_update_status(self) -> None:
 
     @override
     def on_relation_joined(self) -> None:
+        """Any relation event will just share the connection with the client."""
         self.share_connection_info()
 
     @override
     def on_relation_changed(self) -> None:
+        """Any relation event will just share the connection with the client."""
         self.share_connection_info()
 
     @override
     def on_relation_departed(self, departing_unit: Unit | None) -> None:
+        """Any relation event will just share the connection with the client."""
         self.share_connection_info()
```

Comment:
is this true for departed?

Reply:
This is true for departed as well: Departed here is for the peer relation.